pkg/webui/components/icon/index.js

```diff
@@ -61,6 +61,7 @@ const hardcoded = {
   overview: 'dashboard',
   packet_broker: 'camera',
   payload_formats: 'code',
+  recent_actors: 'recent_actors',
```

Comment:
The idea is to map a name that we like to refer to the icon to. Mapping the icon to the same name is a no-op.
```suggestion
  oauth_clients: 'recent_actors',
```
```suggestion
  recent_actors: 'recent_actors',
```